app/enterprise/2.1.x/deployment/installation/amazon-linux-2.md

```diff
@@ -125,7 +125,7 @@ You should now have two files in your home directory on the target Amazon system
 Execute a command similar to the following, using the appropriate RPM file name you downloaded.
 
 ```bash
-$ sudo yum install kong-enterprise-edition-{{page.kong_latest.version}}.aws.rpm
+$ sudo yum install kong-enterprise-edition-{{page.kong_latest.version}}.*.rpm
```

Comment:
What's the rationale in using a wildcard here?

Reply:
hm, good question: we have:

- `kong-enterprise-edition-amzn1.<arch>.rpm`
- `kong-enterprise-edition-amzn2.<arch>.rpm`

where `arch` can be: `amd64` or `noarch` for EE

Reply:
I feel leaving a placeholder for the architecture and listing the possible values for each distro would be a better approach. My question was more about the `aws` infix. Adding the wildcard works, but encourages users to blindly run the command, which will work, but in specific cases might be problematic (e.g., what if there's more than one package downloaded locally?).

Reply:
@gszr but what if there are changes to the architectures or to the possible values for each distro? We would need to remember to update the docs every time. For example, CE has the `aws` infix but EE has the `amzn` infix. We are getting close to deprecating `amzn1`/`aws1` and we could have `arm64` for EE some day too.

Reply:
What if we do something like what Docker does? https://docs.docker.com/engine/install/centos/#install-from-a-package

Just document like this: `sudo yum install /path/to/package.rpm`

Reply:
thoughts here @lena-larionova ?

Reply:
I like Docker's approach, much cleaner. My vote goes towards that option.

Reply:
okay, here it is: https://github.com/Kong/docs.konghq.com/pull/2321/commits/868c6e8e97d0506c20281b88d30f5c3c96ce0020

I also did this: https://github.com/Kong/docs.konghq.com/pull/2321/commits/f110a2be30c5b29a68ebd36019e5a098ab9b389a to ensure CE <> EE have similar install instructions.